static/js/containers/ProductDetailEnrollApp.js

```diff
@@ -66,37 +66,46 @@ export class ProductDetailEnrollApp extends React.Component<
           Enroll
         </ModalHeader>
         <ModalBody>
-          <div className="row modal-subheader">
-            <div className="col-10">Take a course and get a certificate</div>
-            <div className="col-2">${product.price}</div>
+          <div className="row modal-subheader d-flex">
+            <div className="flex-grow-1 align-self-end">Get Certificate</div>
+            <div className="text-right align-self-end">${product.price}</div>
           </div>
           <div className="row">
-            <div className="col-4">
-              <div className="upgrade-icon" />
-            </div>
-            <div className="col-8">
+            <div className="col-12">
+              <p>
+                <span className="font-weight-bold">
+                  Get a certificate today
+                </span>{" "}
+                and lorem ipsum dolor sit amet, consectetur adipiscing elit, sed
+                do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+              </p>
+
               <p>
                 Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
-                eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut
-                enim ad minim veniam, quis nostrud exercitation ullamco laboris
-                nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor
-                in reprehenderit in voluptate velit esse cillum dolore eu fugiat
-                nulla pariatur. Excepteur sint occaecat cupidatat non proident,
-                sunt in culpa qui officia deserunt mollit anim id est laborum.
+                eiusmod tempor incididunt ut labore.
```

Comment:
This copy needs to be updated to match the text quoted in the issue here: https://github.com/mitodl/mitxonline/issues/440

You can disregard the text in the mockups since those haven't been updated.